js_modules/dagit/packages/core/src/pipelines/PipelineReference.tsx

```diff
@@ -24,11 +27,16 @@ export const PipelineReference: React.FC<Props> = ({
   showIcon,
   size = 'normal',
 }) => {
+  const truncatedName =
+    pipelineName.length > TRUNCATION_THRESHOLD
+      ? `${pipelineName.slice(0, TRUNCATION_THRESHOLD - TRUNCATION_BUFFER)}…`
```

Comment:
The idea of the threshold is so that we don't end up just lopping one letter off the end or something. Dunno if five characters is right, maybe three would be better.

Reply:
I like where you're at with 5. Lots of 3 character suffixes, ('ing' and 'est') that would probably come out looking jank if they were truncated by themselves.